icu4c/source/test/intltest/plurults.cpp

```diff
@@ -447,6 +450,129 @@ void PluralRulesTest::testGetSamples() {
   }
 }
 
+void PluralRulesTest::testGetFixedDecimalSamples() {
+  // TODO: fix samples, re-enable this test.
+
+  // no get functional equivalent API in ICU4C, so just
+  // test every locale...
+  UErrorCode status = U_ZERO_ERROR;
+  int32_t numLocales;
+  const Locale* locales = Locale::getAvailableLocales(numLocales);
+
+  FixedDecimal values[1000];
+  for (int32_t i = 0; U_SUCCESS(status) && i < numLocales; ++i) {
+    PluralRules *rules = PluralRules::forLocale(locales[i], status);
+    if (U_FAILURE(status)) {
+      break;
+    }
+    StringEnumeration *keywords = rules->getKeywords(status);
+    if (U_FAILURE(status)) {
+      delete rules;
+      break;
+    }
+    const UnicodeString* keyword;
+    while (NULL != (keyword = keywords->snext(status))) {
+      int32_t count = rules->getSamples(*keyword, values, UPRV_LENGTHOF(values), status);
+      if (U_FAILURE(status)) {
+        errln(UNICODE_STRING_SIMPLE("getSamples() failed for locale ") +
```

Comment:
Instead of `UNICODE_STRING_SIMPLE("...")` use `UnicodeString(u"...")` -- note the `u` prefix.